src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Ball ball = new Ball();
+        Random random = new Random();
+        int num = random.nextInt(100);
```

Comment:
All magic numbers in your code should be constants.
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names

Random and ColorSupplier should be a class-field